zebra/zserv.c

```diff
@@ -67,6 +67,8 @@
 /* privileges */
 extern struct zebra_privs_t zserv_privs;
 
+static struct thread *t_process;
```

Comment:
Doesn't this belong to zebra_router.c?

Reply:
hmm - the zrouter object doesn't appear in zserv, so I guess I didn't think so... the state of the relationships between zebra and its zclients is sort of an aggregate, built up from the various structures that are in zserv.c. are you saying that you'd like me to add the use of zrouter to zserv, and then hang the 'thread' pointer on it?

Reply:
yep.  zrouter should own stuff like this.

Reply:
I did an updated push, and moved event to be per-client, which is really what it should have been (!) see if that looks better to you?